scipy/signal/_multimethods.py

```diff
@@ -0,0 +1,30 @@
+import functools
+import numpy as np
+from scipy._lib.uarray import all_of_type, create_multimethod
+from scipy.signal import _api
+
+
+__all__ = ['upfirdn']
+
+
+_create_signal = functools.partial(create_multimethod,
+                                   domain="numpy.scipy.signal")
+
+
+def _h_x_replacer(args, kwargs, dispatchables):
```

Comment:
Nit: Let's just define the arg replacers directly above the actual function which will use these. Here it should be moved just above `upfirdn`. Just easier to track this way.